src/common/components/layouts/pages/inspect/validationErrorMessages_en.json

```diff
@@ -172,6 +172,83 @@
       "26": {
         "SUMMARY": "Is this a table?",
         "DESCRIPTION": "This text looks like a table but it's tagged as a list. If you intended this to be a table it's better to format it as such."
+      },
+      "27": {
+        "SUMMARY": "Are these different paragraphs?",
+        "DESCRIPTION": "This text is tagged as a single paragraph but appears to be several. Always use paragraph breaks (<Enter>), don't use line breaks (<Shift><Enter>) to delimit paragraphs. In Word you can see paragraph breaks by going to the File - Options - Display menu and selecting Paragraph marks there."
+      },
+      "28": {
+        "SUMMARY": "Is this part of the table of contents?",
+        "DESCRIPTION": "This text appears to be part of the table of contents but is not tagged as such. Always create tables of contents automatically. Don't make them by hand and don't edit them manually."
+      },
+      "29": {
+        "SUMMARY": "Emty table of contents entry",
+        "DESCRIPTION": "This entry in the table of contents appears to be empty. Always create tables of contents automatically. Don't make them by hand and don't edit them manually."
+      },
+      "30": {
+        "SUMMARY": "Table of content entry does not link to any page",
+        "DESCRIPTION": "Make sure table of contents entries link to pages in the document. Always create tables of contents automatically. Don't make them by hand and don't edit them manually."
+      },
+      "31": {
+        "SUMMARY": "Table of contents entry points to the wrong page",
+        "DESCRIPTION": "The page number of this entry does not correspond to the page link. this Always create tables of contents automatically. Don't make them by hand and don't edit them manually."
+      },
+      "32": {
+        "SUMMARY": "Badly aligned table of content entries",
+        "DESCRIPTION": "Table of content entries of the same level appear to be badly aligned. This makes the table of contents less readable. Always create tables of contents automatically. Don't make them by hand and don't edit them manually."
+      },
+      "33": {
+        "SUMMARY": "Table of contents entry not the same as the header it refers to",
+        "DESCRIPTION": "The name of this entry is not the same as the header on the page it refers to. Always create tables of contents automatically. Don't make them by hand and don't edit them manually."
+      },
+      "34": {
+        "SUMMARY": "Is this one list?",
+        "DESCRIPTION": "This looks like one list, but it is tagged as a sequence of shorter lists. If it's meant to be one list, please make sure it is tagged as such. To make a list, always use the list functionality of your office application. Don't format lists manually."
```

Comment:
`it is tagged as a sequence of shorter lists`  It may be not tagged as several lists. It could be several paragraphs, spans, or else structure elements.